### DISCUSSION TITLE
Specify speedtest server for manual/immediate test

### DISCUSSION THREAD

**Discussion by arielzusya** [alexjustesen/speedtest-tracker#1872] [Ideas]
In the config I've included several servers for Ookla to use for its testing. When I see a pattern suggesting a downward trend in speed, I'd like to be able to specify a specific server from that list for retest. That way I can easily manually gather more data to then provide to the ISP when making the case that it wasn't because the server on the other side was slow. The way I see this playing out is, for example, Comcast Business as the ISP and then testing against Xfinity or CenturyLink/Lumen as the ISP and then testing against Quantum rather than just picking one of the servers from the list provided in the config. That way the ISP will have a harder time claiming the issue isn't with them or it is farther downstream. Thanks again for the great work!

**Comment by alexjustesen** [comment]
I think what you're looking for is included in `v0.24.0`, any configured servers will be listed first in the dropdown when you go to manually run a test.

https://github.com/alexjustesen/speedtest-tracker/releases/tag/v0.24.0

**Reply by arielzusya** [reply]
I didn't notice that watchtower hadn't updated to the latest. Just updated and sure enough that feature is there now. Thanks!
